src/constants.ts

```diff
@@ -17,8 +17,10 @@ export enum RPC_METHOD {
 export enum API_TYPE {
   SIGN_TX_URL_RESOLVED,
   RPC_REQUEST,
-  RPC_RESONSE,
+  RPC_RESPONSE,
   RPC_SEND_TO_CONTRACT,
+  GET_INPAGE_QRYPTO_ACCOUNT_VALUES,
+  SEND_INPAGE_QRYPTO_ACCOUNT_VALUES = 'ACCOUNT_CHANGED',
```

Comment:
2 concerns:
1. is there s reason why it needs to be a separate value than the name?
2. lets assign strings for every API_TYPE:
```
SIGN_TX_URL_RESOLVED = 'SIGN_TX_URL_RESOLVED',
RPC_REQUEST = 'RPC_REQUEST',
etc...
```
We shouldn't mix types. The first 5 API types are actually assigned numeric values based on position, SIGN_TX_URL_RESOLVED = 0, RPC_REQUEST = 1, etc. And the last one is assigned a string so lets just make them all strings.